spec/features/user_can_sign_in_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.feature "Session/ Sign-in", type: :feature do
+
+  before(:each) do
+    successful_sign_up
+  end
+
+  scenario "A user can sign in with valid credentials" do
+
+    visit "/session/new"
```

Comment:
The full feature test should probably start by visiting ('/') and then clicking a link on that page which says something like "Already have an account? Click here to log in" (check what facebook actually says) and then redirecting to session/new. Even if it fails for now, let's update the capybara tests to show the complete functionality for now then work out how to implement (we also need to do the same thing on the sign up side (ie. add a redirect from ('/') )

Reply:
replaced   visit "/session/new" with  visit "/"